FinniversKit/Sources/Components/HelthjemView/HelthjemView.swift

```diff
@@ -0,0 +1,219 @@
+import UIKit
+
+public protocol HelthjemViewDelegate: AnyObject {
+    func helthjemViewDidSelectPrimaryButton(_ view: HelthjemView)
+    func helthjemViewDidSelectSecondaryButton(_ view: HelthjemView)
+}
+
+public final class HelthjemView: UIView {
+    public weak var delegate: HelthjemViewDelegate?
+
+    // MARK: - Subviews
+
+    private lazy var imageView: UIImageView = {
+        let image = Config.userInterfaceStyleSupport == .forceLight ?
```

Comment:
hmm, this will fail for `.dynamic`, because we'd always use dark mode icon even if its was dynamic but light 🤔

Reply:
Nice catch, pushed up a change to cater for `.dynamic` type.